features/addressing/managing_countries/managing_provinces_of_country.feature

```diff
@@ -43,6 +43,24 @@ Feature: Managing provinces of a country
         And this country should still have the "Scotland" province
         And this country should still have the "England" province
 
+    @ui @javascript @api
+    Scenario: Removing a province that has been a zone member should be possible
+        Given this country has the "Northern Ireland" province with "GB-NIR" code
+        And this country also has the "Scotland" province with "GB-SCT" code
+        And this country also has the "England" province with "GB-ENG" code
+        And the store has a zone "Great Britain" with code "GB"
+        And it has the "Northern Ireland" province member
+        And it also has the "Scotland" province member
+        And it also has the "England" province member
+        And the "England" province member has been removed from this zone
+        When I want to edit this country
```

Comment:
```suggestion
        When I am editing this country
```

I know we use this step a lot, but is sounds sooo weird :D

Reply:
```suggestion
        And I am editing this country
```
or
```suggestion
        When I edit this country
```
👀